src/routes/home/components/GameListItem.vue

```diff
@@ -55,7 +55,7 @@
 </template>
 
 <script>
-import GameStatus from '../../utils/GameStatus.json';
+import GameStatus  from '_/utils/GameStatus.json';
```

Comment:
nitpick: extra whitespace
```suggestion
import GameStatus from '_/utils/GameStatus.json';
```